src/ontology/mondo-edit.obo

```diff
@@ -357006,9 +357007,7 @@ id: MONDO:0015486
 name: keratoconus
 def: "A degenerative, structural disorder of the eye, characterized by a cone-shaped protrusion of the cornea. It may lead to visual disturbances." [NCIT:P378]
 subset: disease_grouping
-subset: gard_rare {source="GARD:6824", source="MONDO:GARD"}
 subset: otar {source="MONDO:OTAR"}
-subset: rare
```

Comment:
@yshwetar, we do not update the rare disease subset as it is done automatically. 
Also, the annotation says that GARD considers this a rare disease, therefore GARD would have to update this on their side.